recipe/bld.bat

```diff
@@ -6,7 +6,6 @@ cd build
 cmake -G Ninja ^
     -DCMAKE_BUILD_TYPE=Release ^
     -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
-    -DCMAKE_INSTALL_DATADIR=%LIBRARY_PREFIX% ^
```

Comment:
I don't understand the purpose of this option. I removed it to provide consistency between Windows and non-Windows platforms, and [CMake will still be able to find the configs](https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure) (regardless of the `W`/`U` columns, CMake will search these paths on all platforms).

Let me know if I should revert it nevertheless.

Reply:
It tells CMake where to install data artifacts. i.e. man pages, and docs

https://cmake.org/cmake/help/latest/command/install.html#programs

Reply:
You probably shouldn't remove it because our install prefixes on Windows are non-standard, and CMake won't, by default, install things in the correct place. Whereas on unix, $PREFIX/share is a standard place for data artifacts.

Reply:
Change reverted.